### DISCUSSION TITLE
How to completely delete oobabooga and all of it's dependencies (fresh start)

### DISCUSSION THREAD

**Discussion by nb-ai-23** [oobabooga/text-generation-webui#1998] [Q&A]
I have recently run into the error that other people are experiencing with quant-cuda. 

I then uninstalled/resinstalled Cuda, Python and Pytorch, and now it looks like I've broken a few other applications by doing so. I fear this is because I didn't fully uninstalled these and there may be multiple versions of each saved in different places.

Is there a way to start again (fresh start) without reinstalling windows? 

I am relatively new to all of this.

**Comment by sebaxzero** [comment]
if you installed by using the one-click installer then just delete the installer_files and use the start_windows to reinstall everything, this work on its own conda env, so it doesnt mess up other dependencies, when messing with libraries, use first the cmd_windows which activates the env, then in the same cmd do what you need like upgrading libraries.
mind telling exactly what you did and what applications are broken

**Reply by nb-ai-23** [reply]
I understand thank you. When I am installing the UI I get the same as [This](https://github.com/oobabooga/text-generation-webui/issues/1986) then when I try to run the UI after I get a bunch of errors finishing with the line 
"AssertionError: Torch not compiled with CUDA enabled" 

Pretty annoying because this UI used to work for me around a week ago so I must have done something wrong in that time. 

I thought this might have something to do with how I installed CUDA and Python so uninstalled/reinstalled both. Then Stable Diffusion/Auto1111 wouldn't load but I have since fixed the issue with sd/auto1111

Just for reference:
RTX 3090
Win 11
